e2e/jest.e2e.config.js

```diff
@@ -0,0 +1,21 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```

Comment:
Adds type checking to this file! Very nice. This is how `ts-jest` sets it up by default.